aikau/src/main/resources/alfresco/core/Page.js

```diff
@@ -227,7 +228,7 @@ define(["alfresco/core/ProcessWidgets",
        */
       _registerProcessedService: function alfresco_core_Core___registerProcessedService(service, index) {
          this._processedServiceCountdown--;
-         if (index == null || isNaN(index))
+         if (index !== 0 && (!index || isNaN(index)))
```

Comment:
I think this is still not quite right, we still want to push the service if it is at index 0